lib/poller.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const { EventEmitter } = require('events')
+
+class Poller extends EventEmitter {
+  constructor ({
+    pollingRateSeconds,
+    serializeSegments,
+    splitioApi
+  }) {
+    super()
+    this.cache = {}
+    this.pollingRateSeconds = pollingRateSeconds || 300
+    this.serializeSegments = serializeSegments || false
+    this.splitioApi = splitioApi
+    this.timeout = null
+  }
+
+  async pollForChanges () {
+    this.on('poll', async () => {
+      try {
+        const splitChanges = await this.splitioApi.getSplitChanges()
+        delete this.cache.splitChanges
```

Comment:
the deletes aren't totally obvious.  are they necessary?  what's the format of `splitChanges`?